File: app_java/src/main/java/com/dragonforest/demo/app_java/SecondActivity.java
package com.dragonforest.demo.app_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.os.Trace;

import com.dragonforest.demo.app_java.util.PreUtil;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
        initView();
    }

    private void initView() {
        try {
            Thread.sleep(1000);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
    }

    @Override
    protected void onResume() {
        super.onResume();
        PreUtil.INSTANCE.putString(this,PreValues.USER_NAME,"hanlonglin");
        PreUtil.INSTANCE.putString(this,PreValues.USER_PASSWD,"123456");
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
    }
}

File: app/src/main/java/com/dragonforest/demo/systraceplugin/SecondActivity.kt
package com.dragonforest.demo.systraceplugin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.dragonforest.demo.systraceplugin.constants.PreValues
import com.dragonforest.demo.systraceplugin.utils.Monitor
import com.dragonforest.demo.systraceplugin.utils.PreUtil

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
    }

    override fun onResume() {
        PreUtil.putString1(this,PreValues.USER.USER_NAME,"hanlonglin")
        PreUtil.putString1(this,PreValues.USER.USER_PASSWD,"123456")
        super.onResume()
        Monitor.record()
    }
}